### DISCUSSION TITLE
Some SAAS functionality is missing (e.g. Payments/Stripe, Organization/Multi Tenancy)

### DISCUSSION THREAD

**Discussion by canadianeagle** [Dhravya/cloudflare-saas-stack#19] [Ideas]
First of all, 

Great job on this + releasing this to the public for free - very kind of you.
The features above (maybe more), but at minimum, a way to collect payments would make this more complete SAAS kit. I'm guessing PR's are welcome? 

Again, great job. I'll be slowly migrating most of my projects to Cloudflare thanks to a few of your posts on twitter.

Cheers!

**Comment by Dhravya** [comment]
PRs are welcome! I think lmsqueezy/stripe payments will be great
